style.css

```diff
@@ -48,32 +54,218 @@
 }
 
 /* styling the headline introductory message (typography) */
-.greeting-text {
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-}
-
 .text-color {
   display: block;
   color: #36b37e;
 }
 
 /* Styling the supporting Text in the headline section */
-.supportig-text {
+.supportig-text,
+.about-me-intro {
   font-weight: 400;
   font-size: 20px;
   line-height: 28px;
   padding: 9px;
   flex: none;
   flex-grow: 0;
   width: 95%;
+  text-align: center;
 }
 
 .headline-box {
   margin-top: 50px;
-  font-family: "Inter", sans-serif;
-  font-style: normal;
   text-align: center;
   color: #172b4d;
+  align-self: center;
+}
+
+/* Styling for Work Section */
+.my-work {
+  display: grid;
+  grid-template-columns: 320px;
+  justify-content: center;
+  margin: 50px;
+}
+
+.work-intro,
+.about-text,
+.greeting-text {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  width: 340px;
+  color: #172b4d;
+}
+
+.line {
+  border: solid 2px #36b37e;
+  display: flex;
+  border-radius: 24px;
+  justify-content: center;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  width: 48px;
+  height: 4px;
+}
+
+.work-intro-container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.project {
+  border: 2px solid #d0d9d4;
+  display: flex;
+  height: 474px;
+  border-radius: 8px;
+  align-items: flex-end;
+  background-color: #d0d9d4;
+  margin-top: 20px;
+}
+
+.project-title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 16px;
+  width: 316px;
+  height: 222px;
+  background: #fff;
+  border-radius: 8px;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+}
+
+.langues-use {
+  display: flex;
+  list-style: none;
+  gap: 4px;
+}
+
+.language {
+  width: 70px;
+  height: 45px;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  text-align: left;
+  flex: 1;
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 8px 12px;
+}
+
+.ruby {
+  flex: 2;
+}
+
+.project-name {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.see-project,
+.resume {
+  padding: 12px;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  background: #36b37f;
+  border: solid #36b37f;
+  border-radius: 4px;
+  color: #fff;
+}
+
+/* Styling about section */
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  align-items: center;
+}
+
+.sk-logo .fw-logo .lang-logo {
+  width: 80px;
+  height: 80px;
+  background: #fff;
+  box-shadow: 0 16px 24px rgba(37, 47, 137, 0.08);
+}
+
+.list {
+  display: flex;
+  flex-wrap: wrap;
+  height: 180px;
+  list-style: none;
+  align-items: flex-end;
+  justify-content: space-around;
+}
+
+.frame-works,
+.skills,
+.languages {
+  background: #c1c7d0;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+}
```

Comment:
Please, ensure that the background of the cards in the skills section matches the design. Check the above comment for more on why you should stick to the given design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185594054-f9829dea-0d58-41e6-adb5-b786d36a7ec2.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/185594139-0d2c7017-a956-4b72-80ce-bc83f76ebe2b.png)